opm/grid/GraphOfGridWrappers.hpp

```diff
@@ -180,7 +183,7 @@ std::vector<std::vector<int>> communicateExportedWells(
 /// Root is the only rank with information about wells, and communicates
 /// the necessary information to other ranks.
 /// On root ImportList has been already extended with all cells on the current rank.
-void extendExportAndImportLists(const GraphOfGrid<Dune::CpGrid>& gog,
+void extendAndSortExportAndImportLists(const GraphOfGrid<Dune::CpGrid>& gog,
```

Comment:
Please use somewhat consistent indentation.
Usually alignment of the first letter of a line in this file is at the opening brace. Here it is still at the removed brace. In line 173 four spaces seem to be used unconditionally.

I have no idea what our clang-format config file would use, though.

Reply:
clang-format does not align second and following parameters to the opening brace but indents them with 4 spaces. It looks bad if `[[maybe_unused]]` is present.